zeppelin-interpreter/src/main/java/org/apache/zeppelin/cluster/ClusterStateMachine.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.cluster;
+
+import io.atomix.copycat.server.Commit;
+import io.atomix.copycat.server.Snapshottable;
+import io.atomix.copycat.server.StateMachine;
+import io.atomix.copycat.server.StateMachineExecutor;
+import io.atomix.copycat.server.storage.snapshot.SnapshotReader;
+import io.atomix.copycat.server.storage.snapshot.SnapshotWriter;
+import org.apache.zeppelin.cluster.meta.ClusterMeta;
+
+/**
+ * Cluster State Machine
+ */
+public class ClusterStateMachine extends StateMachine implements Snapshottable {
+  private ClusterMeta clusterMeta = new ClusterMeta();
+
+  @Override
+  protected void configure(StateMachineExecutor executor) {
+    executor.register(PutCommand.class, this::put);
+    executor.register(GetQuery.class, this::get);
```

Comment:
better to name it GetCommand to make them consistent.

Reply:
Even if there are two types, ppl might not think `GetCommand` changes something. So I agree with @zjffdu